packages/core/src/transforms/resize.ts

```diff
@@ -91,8 +91,8 @@ export const resize: TransformFactory<ResizeOptions> = (config) => {
       finalWidth = originalWidth
       finalAspect = originalAspect
 
-      console.warn(
-        '[vite-imagetools] withoutEnlargement or withoutReduction enabled. Image width, height and aspect ratio reverted to original values'
+      context.warn(
```

Comment:
This warning doesn't seem very useful. E.g. it doesn't tell you what image it encountered

I also wonder if this should even be a warning? This doesn't seem like an error condition, but just regular functioning. Maybe it should be something more like `debug` or `info`. I'm not that familiar with the feature though. @sawyerclick can you provide some more details?

Reply:
@JonasKruckenberg recommended the warning as a part of his review in the original pr #461

Now that I'm looking at it again, I don't believe the warning is needed. It's redundant as it's sole purpose is to indicate that the flags are working.

Reply:
Ah, I see: https://github.com/JonasKruckenberg/imagetools/pull/461#discussion_r1060480418

@JonasKruckenberg it seems this warning has been a bit noisy. Do you mind if we either remove it or demote it to an `info` or `debug`? I think if users need to figure out whether it was triggered they can always compare the dimensions on the filesystem to the rendered dimensions

Reply:
My concern was that the whole idea of `without*` is to not do anything when the conditions are met. To the uninitiated (or tired) this might seem like the whole plugin not doing it's job though. Also in all cases I can imagine `without*` is a safeguard that trips when you made an error while specifying your image dimensions. That's why I originally recommended the warning, but I guess `info` makes sense too (since it technically works as intended)